specification/app/resource-manager/readme.md

```diff
@@ -65,11 +65,17 @@ input-file:
   - Microsoft.App/preview/2024-02-02-preview/Subscriptions.json
   - Microsoft.App/preview/2024-02-02-preview/Usages.json
   - Microsoft.App/preview/2024-02-02-preview/FunctionsExtension.json
+  - Microsoft.App/preview/2024-02-02-preview/SessionPools.json
+  - Microsoft.App/preview/2024-02-02-preview/Sessions.json
 directive:
   - suppress: PatchBodyParametersSchema
     from: JavaComponents.json
     reason: |
       Java Component is using componentType as the discriminator. While the discriminator is a required property, this rule prevent it being present in the patch request body.
+  - suppress: LroErrorContent
```

Comment:
what is the error related to this?

Reply:
Hi, I removed the suppression temporary, and there would be lint error now https://github.com/Azure/azure-rest-api-specs/pull/28047/checks?check_run_id=24182704181
![image](https://github.com/Azure/azure-rest-api-specs/assets/101094705/dd2517bc-eeb6-46a5-b368-f37b03c94cb4)